tools/build

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Rudimentary safety check
+[[ ! -f "tools/build" ]] && echo 'Please run this from package root!' && return 1;
```

Comment:
Indeed, this could work in some instances, but would affect chaining commands, given that you'd be changing the working dir:

`tools/ $ . clean && . svg-optimize`

Alternatively we could make these executable (I wasn't sure if there was precedent in other repos, but I see now there are some scripts that are like that). I'll consider changing them to be executables.